kivy/core/text/text_layout.pyx

```diff
@@ -565,6 +565,7 @@ def layout_text(object text, list lines, tuple size, tuple text_size,
     # above, we might have gone a few lines over
     if max_lines > 0 and len(lines) > max_lines:
         val = True
+        h -= lines[0].h * (len(lines) - max_lines)
```

Comment:
Shouldn't this be `if ...: h-= sum(l.h for l in lines[max_lines:])` as opposed to just using the first line's height?

Reply:
Each line have a different height?
If yes, I missed something, so your advice is correct.

Reply:
Yeah, You were right.
If the user is using for example a markup, We need to handle this kind of situation.
![Screenshot from 2019-09-13 21-12-40](https://user-images.githubusercontent.com/8177736/64888610-b51e0900-d66b-11e9-9e16-e0038911022b.png)
I've just committed the fix you proposed.